Dockerfile

```diff
@@ -25,7 +25,7 @@ ENV MLSERVER_MODELS_DIR=/mnt/models \
 RUN apt-get update && \
     apt-get -y --no-install-recommends install \
         unattended-upgrades \
-        libgomp1 libgl1-mesa-dev libglib2.0-0 build-essential && \
+        libgomp1 libgl1-mesa-dev libglib2.0-0 build-essential ffmpeg && \
```

Comment:
We have now moved to a different base image for Docker, but it should already install ffmpeg (although in a diff way). Therefore, feel free to pick up `master`'s version of the `Dockerfile` when rebasing :+1: